src/TutorProxy.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace ETNA\Silex\Provider\TutorProxy;
+
+use GuzzleHttp\Client;
+
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+
+class TutorProxy implements ServiceProviderInterface
+{
+    private $controller_instance = null;
+
+    public function __construct($controller_instance = null)
+    {
+        $this->controller_instance = $controller_instance;
```

Comment:
est-ce que pour le moment on s'en fout pas de ça, et que si une personne veut ajouter ces propres controllers, il le rajoutera à ce moment là?

Reply:
pas trop parce que j'ai besoin d'un cnstructeur car, ds la fonction register plus bas j'ai besoin de faire un `$app->mount("/", $this->controller_instance);`